doc/introduction.rst

```diff
@@ -79,6 +79,34 @@ necessarily the identity of indiscernibles.
   parameterizations are equivalent. In practice, an algorithm may thus solve
   the metric learning problem with respect to either :math:`M` or :math:`L`.
 
+.. _bilinear_similarity:
+
+Bilinear Similarity
+===================
+
+Some algorithms in the package don't learn a distance or pseudo-distance, but
+a similarity. The idea is that two pairs are closer if their similarity value
+is high, and viceversa. Given a real-valued parameter matrix :math:`W` of shape
```

Comment:
"Some algorithms in the package learn bilinear similarity functions. These similarity functions are not pseudo-distances: they simply output real values such that the larger the similarity value, the more similar the two examples.

Reply:
Changed